services/micrologproxyhub/grpcapi/grpcapi.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package micrologproxyhubgrcpapi;
```

Comment:
should be `micrologproxyhubgrpcapi`, personally I like having dots in the name, easier to read

Reply:
dots converts to underline in the pb-file, and thats breaks the convension from golang as the package name shouldn't contains underlines. Read more about at: [https://blog.golang.org/package-names](https://blog.golang.org/package-names)